plugins/minigames.json

```diff
@@ -919,17 +919,23 @@
         }
       }
     },
-    "super_duel": {
+    "duel": {
       "description": "Crush your enemy in a 1v1",
       "external_url": "https://www.youtube.com/watch?v=hvRtiXR-oC0",
       "authors": [
         {
-          "name": "JoseAng3l",
+          "name": "Unknown/JoseAng3l",
```

Comment:
let's not add Unknown here since the initial version of this minigame we've had here has been for 1.6 or 1.7 which was made by  JoseAng3l i think(?)